Wikipedia/Code/EditSaveViewController.swift

```diff
@@ -161,7 +163,7 @@ class EditSaveViewController: WMFScrollViewController, Themeable, UITextFieldDel
 
         setupButtonsAndTitle()
         mode = .preview
-        
+        setupSemanticContentAttibute()
```

Comment:
I notice the fine print text views at the bottom don't seem to take effect for RTL (HE example):

![IMG_0161](https://user-images.githubusercontent.com/3620196/216680023-39018877-4722-4621-8bfa-c8e0e9795f05.PNG)

I think the alignment setting becomes overwritten any time the attributed strings are updated when `updateTextViews()` is called, which is called in a couple of different places. When I move this `setupSemanticContentAttribute()` call to the end of the `updateTextViews` method, that seems to fix it. I'm open to other options though!

Reply:
Sounds good to me!